libc/test.yaml

```diff
@@ -1,10 +1,35 @@
 test:
   type: unity
   tests:
-    - name: unit
-      execute: test-libc
-      #FIXME: symlink tests will fail on non-rootfs targets
-      #FIXME: unistd_uid doesn't work on zynq7000-qemu
+    - name: printf
+      execute: test-libc-printf
       targets:
         exclude: [armv7m4-stm32l4x6-nucleo]
         include: [host-generic-pc]
+
+    # Executing test-printf on 'armv7m4-stm32l4x6-nucleo' takes ~9 min, that's why we want to run it only during nightly campaigns 
```

Comment:
uh, could You elaborate what takes so much time at this target (i would like to be sure it's not a problem with `printf` implementation)

Reply:
Of course, I'm almost certain that it's not the fault of the printf implementation, the test itself is quite extensive, but I will try to check as soon as possible what exactly causes this test to take so long to run on stm32l4x6-nucleo.